src/components/TimeseriesChartExport/stories/full.md

```diff
@@ -0,0 +1,129 @@
+# Timeseries Chart Export
+
+<!-- STORY -->
+
+#### Description:
+
+Component allows to export CSV data for provided timeseries in selected timerange with defined granularity.
+
+Component based on antd modal component with form, where user can define all configurations for CSV export.
+
+Component has embedded logic for fetching timeseries data and generating CSV file. To make it work properly, component
+has to be wrapped into `ClientSDKProvider` component with defined client SDK instance. Fetching logic
+also can be redefined via `retrieveTimeseries` and `fetchCSV` props.
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { TimeseriesChartExport } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+  return (
+    <TimeseriesChartExport
+      visible={true}
+      timeseries={['VAL_23-PDT-96167:X.Value']}
+      granularity={'2m'}
+      defaultRange={[1567321800000, 1567408200000]}
+    />
+  );
+
+}
+```
+
+#### Available props:
+
+##### Required:
+
+| Property       | Description                                                                           | Type                   | Default |
+| -------------- | ------------------------------------------------------------------------------------- | ---------------------- | ------- |
+| `timeseries`   | Array of timeseries ids or externalIds                                                | `(number \| string)[]` |         |
+| `granularity`  | String, that represents initial granularity (ex. 2m, 15s, 1h) to be displayed in form | `string`               |         |
+| `defaultRange` | Array with start - end values for initial time range                                  | `number[]`             |         |
+| `visible`      | Flag that shows/hides modal with form                                                 | `boolean`              |         |
+
+##### Optionals:
+
+| Property             | Description                                                                                                                   | Type                  | Default      |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------- | --------------------- | ------------ |
+| `modalWidth`         | Width of modal window                                                                                                         | `number`              | `600`        |
```

Comment:
not sure what's the best practice here, but can we wrap it in `styles` object prop or something? to bee able to extend it if we have similar props later?
what do you think? or maybe styles is not correct wording, idk

Reply:
for now it should be just a width of modal, cause it pass the value directly to the modal prop. If users will require a styling – we define approach then.